dds_cli/project_status.py

```diff
@@ -102,6 +107,29 @@ def update_status(self, new_status, deadline=None, is_aborted=False, no_mail=Fal
         if is_aborted:
             extra_params["is_aborted"] = is_aborted
 
+        # If the status is going to be archived or deleted. Ask for confirmation
+        if new_status in ["Archived", "Deleted"]:
+            # get project info
+            project_info = self.get_project_info()
+            table = self.generate_project_table(project_info=project_info)
+            dds_cli.utils.console.print(table)
+
+            # Create confirmation prompt
+            print_info = (
+                f"Are you sure you want to modify the status of {self.project}? All its contents "
+            )
+            if new_status == "Deleted":
+                print_info += "and metainfo "
+            print_info += (
+                "will be deleted!\n" f"The project {self.project} is about to be {new_status}.\n"
+            )
+
+            dds_cli.utils.console.print(print_info)
```

Comment:
I liked how you had colored certain things before my last review - could you add so that
- `{self.project}` is `'{self.project}'`
- `{new_status}` is bold and blue like you had it before

This also helped. Other than that - this looks great.